src/main.rs

```diff
@@ -123,11 +123,30 @@ impl State {
     /// Initialize the window and its EGL surface.
     fn init_window(&mut self, connection: &mut Connection, queue: &QueueHandle<Self>) {
         // Initialize EGL context.
-        let native_display = NativeDisplay::new(connection.display());
-        let display = EGLDisplay::new(&native_display, None).expect("Unable to create EGL display");
-        let context =
-            EGLContext::new_with_config(&display, GL_ATTRIBUTES, Default::default(), None)
-                .expect("Unable to create EGL context");
+        let mut raw_display = WaylandDisplayHandle::empty();
+        raw_display.display = connection.backend().display_ptr().cast();
+        let raw_display = RawDisplayHandle::Wayland(raw_display);
```

Comment:
This is more of a `raw_window_handle` issue than a Glutin issue, but this seems like a pretty roundabout way to acquire a `RawDisplayHandle::Wayland`.

Reply:
I guess.